### PR TITLE
[OSC-1291] add schema revision tool alembic

### PR DESCRIPTION
**Pull request by seanbudd**
https://pageuppeople.atlassian.net/browse/OSC-1291

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,10 +1,14 @@
 from setuptools import setup, find_packages
 
 setup(name='dpo',
-      version='0.0.2',
+      version='0.1.0',
```
Comment (seanbudd) [inline setup.py]:
```suggestion
      version='0.1.1',
```

bump version number up for next release

File: setup.py [hunk 2]
```diff
@@ -1,10 +1,14 @@
 from setuptools import setup, find_packages
```
Comment (waltaro) [inline setup.py]: Is this file still required granted we're using pipenv to manage dependencies?

File: dpo/alembic/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Ensure any new tables inherit from the same Base used in `alembic/env.py`
+
+```python
+from dpo.Shared import BaseEntity
+```
+
+Whenever you make a schema change, run
+
+```bash
+pip install .
```
Comment (seanbudd) [inline dpo/alembic/README.md]:
```suggestion
pipenv install .
```

[the remark this answers is not in the document]
Comment (seanbudd) [reply dpo/alembic/README.md]: pipenv, fixed it

File: setup.py (hunk 2 above)
Comment (seanbudd) [reply setup.py]: yeah it's required whenever anything is installing the package. I believe this tells pipenv what's needed and pipenv is just a consumer of this?

Comment (waltaro) [reply setup.py]: Pipenv already has a file that it reads from to install dependencies -- `Pipfile`.. so I don't believe that is the case?
